### ISSUE TITLE
Crashing immediately

### ISSUE THREAD

**Issue by gorgonz** [ita1024/semantik#71]
I'm not sure, whether this is the right location or not. The bug report sent me to http://code.google.com/p/semantik/issues/list, where I was forwarded to here.

Reproduce it with
- opensuse leap 42.2
- install from the official OSS repository (0.9.4-3.33)
- start semantik
- Right click into the yellow layout pane and select "Add a new element"

The application will crash

Could not attach the backtrace file, so here the content:
```
Application: semantik (0.9.4)
KDE Platform Version: 4.14.25
Qt Version: 4.8.6
Operating System: Linux 4.4.74-18.20-default x86_64
Distribution: "openSUSE Leap 42.2"

-- Information about the crash:
  - Das Programm starten
  - Im gelben Layoutbereich rechte Maustaste drücken und "Add an Element" klicken
  ==> Absturz

The crash can be reproduced every time.

-- Backtrace:
Application: Semantik (semantik), signal: Segmentation fault
Using host libthread_db library "/lib64/libthread_db.so.1".
[KCrash Handler]
#6  0x00007f2ad89cb7fd in mem_sel::mem_sel(sem_mediator*) () at /usr/lib64/libsemantik.so.0
#7  0x00007f2ad89cb93c in mem_add::init() () at /usr/lib64/libsemantik.so.0
#8  0x000000000043cc2d in canvas_view::slot_add_item() ()
#9  0x00007f2ad86380ea in QMetaObject::activate(QObject*, QMetaObject const*, int, void**) () at /usr/lib64/libQtCore.so.4
#10 0x00007f2ad79b85b2 in QAction::triggered(bool) () at /usr/lib64/libQtGui.so.4
#11 0x00007f2ad79b9f63 in QAction::activate(QAction::ActionEvent) () at /usr/lib64/libQtGui.so.4
#12 0x00007f2ad7de32d9 in  () at /usr/lib64/libQtGui.so.4
#13 0x00007f2ad7de7829 in  () at /usr/lib64/libQtGui.so.4
#14 0x00007f2ad7a0dbba in QWidget::event(QEvent*) () at /usr/lib64/libQtGui.so.4
#15 0x00007f2ad7deb67b in QMenu::event(QEvent*) () at /usr/lib64/libQtGui.so.4
#16 0x00007f2ad79be6fc in QApplicationPrivate::notify_helper(QObject*, QEvent*) () at /usr/lib64/libQtGui.so.4
#17 0x00007f2ad79c4d7a in QApplication::notify(QObject*, QEvent*) () at /usr/lib64/libQtGui.so.4
#18 0x00007f2ad44a7e3a in KApplication::notify(QObject*, QEvent*) () at /usr/lib64/libkdeui.so.5
#19 0x00007f2ad862417d in QCoreApplication::notifyInternal(QObject*, QEvent*) () at /usr/lib64/libQtCore.so.4
#20 0x00007f2ad79c4573 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool) () at /usr/lib64/libQtGui.so.4
#21 0x00007f2ad7a35f7c in  () at /usr/lib64/libQtGui.so.4
#22 0x00007f2ad7a346bc in QApplication::x11ProcessEvent(_XEvent*) () at /usr/lib64/libQtGui.so.4
#23 0x00007f2ad7a5b552 in  () at /usr/lib64/libQtGui.so.4
#24 0x00007f2ad26e3134 in g_main_context_dispatch () at /usr/lib64/libglib-2.0.so.0
#25 0x00007f2ad26e3388 in  () at /usr/lib64/libglib-2.0.so.0
#26 0x00007f2ad26e342c in g_main_context_iteration () at /usr/lib64/libglib-2.0.so.0
#27 0x00007f2ad8650fae in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /usr/lib64/libQtCore.so.4
#28 0x00007f2ad7a5b606 in  () at /usr/lib64/libQtGui.so.4
#29 0x00007f2ad8622d3f in QEventLoop::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /usr/lib64/libQtCore.so.4
#30 0x00007f2ad8623035 in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () at /usr/lib64/libQtCore.so.4
#31 0x00007f2ad8628489 in QCoreApplication::exec() () at /usr/lib64/libQtCore.so.4
#32 0x000000000042ee3a in main ()

An http://code.google.com/p/semantik/issues/list berichten

```

**Comment by ita1024** [comment]
That bug was fixed in some other version. Try double-clicking instead.

A new release will be made soon.

**Comment by gorgonz** [comment]
@ita1024 confirmed, the workaround with a double-click works fine

**Comment by gorgonz** [comment]
@ita1024 Now I made to build a version from master. It was successful and this version is labeled as "1.0". And good news: It works :-)
Still I have the problem with "empty" as only possible text. Maybe it is a problem of my handling. Is there an advice for me?

**Comment by ita1024** [comment]
To replace the text, select the item and then press the "Enter" key on your keyboard.

**Comment by gorgonz** [comment]
@ita1024 ENTER, oh man, it is so simple, but i did not get it ;-). Just a suggestion for semantik dummies like me: If You offer a right click option "replace text" and add the shortcut hint "ENTER", then users like me get a chance :-)

**Comment by ita1024** [comment]
The problem was fixed a long time ago (0.9.5), but there is semantik 1.0.0 out there.
